kivy/tests/test_app.py

```diff
@@ -30,6 +31,19 @@ def test_user_data_dir(self):
         if not os.path.exists(data_dir):
             raise Exception("user_data_dir does not exist")
 
+    def test_directory(self):
+        a = App()
+        if not os.path.exists(a.directory):
+            raise Exception("directory does not exist")
+
+    def test_name(self):
+        class NameTest(App):
+            pass
+
+        a = NameTest()
+        if a.name != 'nametest':
```

Comment:
why isn't this and the above test simply and `assert` (`assert a.name == 'nametest'`)?

Reply:
Good question. No idea :-) Changed, thanks.